fdbserver/storageserver.actor.cpp

```diff
@@ -9149,6 +9150,18 @@ ACTOR Future<Void> storageServerCore(StorageServer* self, StorageServerInterface
 			when(FetchCheckpointRequest req = waitNext(ssi.fetchCheckpoint.getFuture())) {
 				self->actors.add(fetchCheckpointQ(self, req));
 			}
+			when(UpdateCommitCostRequest req = waitNext(ssi.updateCommitCostRequest.getFuture())) {
+				TraceEvent("BusiestWriteTag", self->thisServerID)
```

Comment:
This probably isn't a huge deal, but I think this might be prone to multiple deliveries, in which case this could be logged multiple times for one message. If that happens, it could interfere with generating statistics from these events.

You could probably avoid that with some extra fields that help you sequence the events.

Reply:
Not sure about this, but would `trackLatest` ensures the latest event get sampled?